nodejs/webapi/swagger/openapiservice.js

```diff
@@ -0,0 +1,63 @@
+const port = 3030;
```

Comment:
Can we also use `const port = process.env.PORT || 3030;`, so that we have flexibility to reference the `env.PORT` value?

Reply:
Moved port options to config.js